tmt/steps/provision/connect.py

```diff
@@ -45,7 +45,7 @@ def options(cls, how=None):
         """ Prepare command line options for connect """
         return [
             click.option(
-                '-g', '--guest', metavar='GUEST',
+                '-g', '--guest', metavar='GUEST', required=True,
```

Comment:
As discussed on IRC, it's possible to override a config stored in a plan, for example:

```yaml
provision:
    how: connect
    guest: my.test.box
execute:
    script: tmt --help
```

Then overriding some of the parameters from the command line, e.g.

    tmt run -avvvddd provision -h connect --port 123

In that case the `--guest` option is not needed/desired.

Reply:
Good point and great catch! Ok. Let's try it again without required.

Reply:
`tmt -vvvddd run provision -h connect`
now returns:

...
Provide a host name or an ip address to connect.


Which seems to be right.